app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,103 @@
+<section class="detail-container">
+
+<h1 class="recipe-name"><%= @recipe.name %></h1>
+<div class="recipe-info">
+  <div class="recipe-info-item1">
+    <div class="desc">
+        <p>Preparation Time: <%= @recipe.preparation %></p>
+        <p>Cooking Time: <%= @recipe.cooking_time %> hours</p>
+        <p>Steps go here...</p>
+    </div>
+    <div class="toggle">
+   <%= button_to "Toggle", toggle_public_recipe_path(@recipe), method: :post, class: "toggle-btn hide-text" %>
+</div>
+
+  
+  </div>
+  <div class="recipe-info-itemBtn">
+    <%= link_to "Generate Shopping List", "#", class: "select-inventory add-ingredient", data: { toggle: "modal", target: "#inventoryModal" } %>
+   <%= link_to 'Add Ingredients', new_recipe_recipe_food_path(@recipe) class: "add-ingredient" %>
+  
+</div>
+
+<table class="recipe-table">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% @recipe_foods.each do |recipe_food| %>
+    <tr>
+      <td><%= recipe_food.food.name %></td>
+      <td><%= recipe_food.quantity %></td>
+      <td><%= recipe_food.food.value * recipe_food.quantity if recipe_food.food.value %></td>
+
+      <td>
+        <%= link_to "Modify", edit_recipe_recipe_food_path(@recipe, recipe_food), class: "modify-btn" %>
+        <%= link_to "Remove", recipe_recipe_food_path(@recipe, recipe_food), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger" %>
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix the `adding recipe` function, as when I try to add a recipe I get the following error
![image](https://github.com/bill7pearl/recipe-rails/assets/81932478/e91b9511-66fd-47eb-8dac-a215b52d3c27)
- [ ] Therefore I am unable to see the recipes list, which should follow this given [design](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_index.png?token=GHSAT0AAAAAACCR4VL54N2AUWYNHDRWMPEKZDAA64Q)

### _Note that this error also occurs when creating public recipes, even though  after refreshing the page public recipes are being added_